tests/utils/test_ensemble.py

```diff
@@ -101,6 +103,26 @@ def create_graphSAGE_model(graph, link_prediction=False):
     return base_model, keras_model, generator, train_gen
 
 
+def create_Unsupervised_graphSAGE_model(graph):
```

Comment:
For consistency, might want to rename `create_Unsupervised_graphSAGE_model` as `create_unsupervised_GraphSAGE_model`; minor point.

Reply:
Done.